Userland/Applications/3DFileViewer/WavefrontOBJLoader.cpp

```diff
@@ -20,9 +20,7 @@ RefPtr<Mesh> WavefrontOBJLoader::load(Core::File& file)
     dbgln("Wavefront: Loading {}...", file.name());
 
     // Start reading file line by line
-    for (auto line = file.line_begin(); !line.at_end(); ++line) {
-        auto object_line = *line;
-
+    for (auto object_line : file.lines()) {
```

Comment:
Shouldn't `object_line` be by reference, so we don't needlessly copy a string?

Reply:
It shouldn't, the line iterators actually return `AK::StringView`s into an internal buffer (that are valid until the iterator is incremented), meaning no copies are made. Since the StringView being returned is a temporary, you can iterate by const reference but not by reference.

Reply:
Thanks, missed that.